README.md

```diff
@@ -331,6 +332,22 @@ modifyWebpackConfig: (webpackConfig, settings) => {
 };
 ```
 
+## Upgrading
+
+There are currently 2 different ways you can upgrade your current project to the newest version of `@availity/workflow`.
+
+### @availity/upgrader
+You can install the availity uprader package manually in run the upgrade script that way.
+```
+npm install @availity/upgrader && ./node_modules/.bin/upgrade-workflow
```

Comment:
Can we change the of the package to `@availity/workflow-upgrade`? I think makes it a bit more clear the intent of the package.

Also, probably better not to ask users to install this as a `devDependency`. In usage notes use:

```
npx @availity/workflow-upgrade
```

Also, let's give credit to https://github.com/typicode/husky/blob/master/src/upgrader in the README.

One thing that I like about `babel-upgrade` is that unless the `--write` flag is passed in cli the upgrader will show a diff https://github.com/babel/babel-upgrade/blob/master/src/index.js#L90 (this might be overkill for now but useful in the future)

Reply:
Sounds good just committed the changes